x-pack/legacy/plugins/apm/server/lib/service_map/service-connection-es-scripts.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const initServiceConnsScript = {
+  lang: 'painless',
+  source: `
+    state.mappedDocs = new HashMap();
+    String[] docKeys = new String[] {
+      '@timestamp',
+      'transaction.id',
+      'parent.id',
+      'span.id',
+      'span.type',
+      'span.subtype',
+      'service.name',
+      'service.environment',
+      'destination.address'
```

Comment:
What do you think about using constants here?

Reply:
I considered it but felt it was best to avoid interpolation of the script source. I didn't want it to look like i was generating dynamic scripts from our code.